swath_projector/utilities.py

```diff
@@ -216,3 +225,24 @@ def make_array_two_dimensional(one_dimensional_array: np.ndarray) -> np.ndarray:
 
     """
     return np.expand_dims(one_dimensional_array, 1)
+
+
+def transpose_if_xdim_less_than_ydim(variable: np.ndarray) -> np.ndarray:
+    """Return transposed variable when variable is wider than tall.
+
+    QuickFix (DAS-2216): We presume that a swath has more rows than columns and
+    if that's not the case we transpose it so that it does.
+    """
+    if variable.ndim != 2:
+        raise ValueError(
+            f'Input variable must be 2 dimensional, but got {variable.ndim} dimensions.'
+        )
+    if variable.shape[0] < variable.shape[1]:
+        transposed_variable = np.ma.transpose(variable).copy()
+        if hasattr(variable, 'mask'):
+            transposed_variable.mask = np.ma.transpose(variable[:].mask)
+            return transposed_variable
```

Comment:
couldn't this be deleted and just let the var be returned on L246?

Reply:
Fixed in commit [735894d](https://github.com/nasa/harmony-swath-projector/pull/19/commits/735894d1f7c3771c9aa53047e64669af8db5f877)